charts/operator/values.yaml

```diff
@@ -39,3 +39,75 @@ affinity: {}
 
 # Tolerations for operator pods
 tolerations: []
+
+# Settings for kube-state-metrics
+metrics:
+  enabled: true
+  # Configuration for the dashboard
+  dashboard:
+    enabled: false
+    labels:
+      grafana_dashboard: "1"
+  prometheus:
+    rules:
+      enabled: true
+    monitor:
+      enabled: true
+      honorLabels: true
+  # Disable all the default collectors
+  collectors: []
+  # Allow kube-state-metrics read-only access to our CRDs
+  rbac:
+    create: true
+    extraRules:
+      - apiGroups:
+          - caas.azimuth.stackhpc.com
+        resources:
+          - clusters
+          - clustertypes
+        verbs:
+          - list
+          - watch
+          - get
+  # Configure kube-state-metrics to report only on our custom resources
+  extraArgs:
+    - --custom-resource-state-only=true
+  customResourceState:
+    enabled: true
+    config:
+      kind: CustomResourceStateMetrics
+      spec:
+        resources:
+          - groupVersionKind:
+              group: caas.azimuth.stackhpc.com
+              version: v1alpha1
+              kind: Clusters
+            metricNamePrefix: azimuth_caas_clusters
+            labelsFromPath:
+              realm_namespace: [metadata, namespace]
+              realm_name: [metadata, name]
+            metrics:
+              - name: phase
+                help: "Cluster phase"
+                each:
+                  type: Info
+                  info:
+                    labelsFromPath:
+                      phase: [status, phase]
```

Comment:
Are there other metrics you want for the clusters? It would be nice to do things like number of clusters by type?

Reply:
yeah, I just went simple to start with, but group by type is a good idea